_episodes/02-filedir.md

```diff
@@ -41,7 +41,7 @@ First let's find out where we are by running a command called `pwd`
 (which stands for "print working directory"). Directories are like *places* - at any time
 while we are using the shell we are in exactly one place, called
 our **current working directory**. Commands mostly read and write files in the 
-current working directory, i.e. "here", so knowing where you are before running
+current working directory (i.e. "here") so knowing where you are, before running
```

Comment:
The brackets are fine, but the comma below obstructs the flow of the sentence.
>so knowing where you are,

How about 
> Commands mostly read and write files in the 
current working directory (i.e. "here"), so knowing where you are before running
a command is important.

Reply:
I agree